dockers/docker-fpm-quagga/bgpcfgd

```diff
@@ -93,6 +125,8 @@ class BGPConfigManager(object):
         daemon.add_manager(swsscommon.CONFIG_DB, swsscommon.CFG_DEVICE_NEIGHBOR_METADATA_TABLE_NAME, self.__neighbor_metadata_handler)
         daemon.add_manager(swsscommon.CONFIG_DB, swsscommon.CFG_BGP_NEIGHBOR_TABLE_NAME, self.__bgp_handler)
         daemon.add_manager(swsscommon.STATE_DB, swsscommon.STATE_INTERFACE_TABLE_NAME, self.__if_handler)
+        if g_allow_list_enabled:
+            daemon.add_manager(swsscommon.CONFIG_DB, "BGP_ALLOWED_PREFIXES", self.__bgp_allowed_handler)
```

Comment:
Pavel,

Introducing new schema for prefixes specific to BGP might confuse the users while the generic schema (IP_PREFIX) is already defined in the below pull request, any thoughts?

https://github.com/Azure/SONiC/blob/48e9012c548528b6528745bda9d75b4164e785eb/doc/mgmt/SONiC_Design_Doc_Unified_FRR_Mgmt_Interface.md